nilearn/glm/first_level/first_level.py

```diff
@@ -1015,6 +1017,86 @@ def first_level_from_bids(
     The subject label of the model will be determined directly
     from the :term:`BIDS` dataset.
 
+    kwargs: :obj:`dict`
+
+        .. versionadded:: 0.11.0
+
+    Keyword arguments to be passed to functions called within this function.
+
+    Kwargs prefixed with ``confound_``
+    will be passed to :func:`~nilearn.interfaces.fmriprep.load_confounds`.
+    This allows to ``first_level_from_bids`` to return
+    a specific set of confounds by relying confound loading strategies
+    defined in :func:`~nilearn.interfaces.fmriprep.load_confounds`.
+    If no kwargs are passed, ``first_level_from_bids`` will return
+    all the confounds available in the confounds TSV files.
+
+    Examples
+    --------
+    If you want to only load
+    the rotation and translation motion parameters confounds:
+
+    .. code-block:: python
+
+        models, m_imgs, m_events, m_confounds = first_level_from_bids(
```

Comment:
```suggestion
        models, imgs, events, confounds = first_level_from_bids(
``` (not sure what the `m_` means)

Reply:
That was a copy-pasta from the code we have in the tests: probably could change it there too.

Reply:
FYI those were shorter form of the name of the return argument:
- models_run_imgs
- models_events
- models_confounds

Reply:
I would prefer  `models, imgs, events, confounds = first_level_from_bids`

Reply:
@bthirion I renamed those variable in the doc strings.

they also appear like this in a few tests, I could do a bit of renaming there too just for internal consistency.